lib/chef/resource/windows_certificate.rb

```diff
@@ -87,6 +87,10 @@ class WindowsCertificate < Chef::Resource
         description: "Ensure that sensitive resource data is not logged by the #{ChefUtils::Dist::Infra::CLIENT}.",
         default: lazy { pfx_password ? true : false }, skip_docs: true
 
+      property :exportable, [TrueClass, FalseClass],
```

Comment:
This needs the introduced field so the documentation will be correctly built

Reply:
Added